infrastructure/removePublicGCS/README.md

```diff
@@ -0,0 +1,34 @@
+# removePublicGCS - Sample 5-minute Solution to prevent public objects in GCS via Cloud Functions
+
+This is a sample of deploying a *Cloud Function* that triggers off a target *GCS* bucket whenever an object's metadata or ACLs are updated. For each object that is updated, this function will explicitly remove the **'allAuthenticatedUsers'** and **'allUsers'** special ACL groups from the owners, readers, and writers groups. This function effectively reacts and removes public access if it is ever accidentally granted by a user or code. The object getting the public ACLs removed are not modified in any other way, the object data, metadata, and other ACLs will remain the same after the function triggers.
+
+## Steps
```

Comment:
You'll need something in your steps that calls out our docs about Service Accounts for Cloud Functions, as well as the GCS roles that the SA will need to do this work. Including a gcloud command that tests the IAM permissions for modifying an object on the target bucket would really help with troubleshooting as well.